modules/ROOT/pages/migration/index.adoc

```diff
@@ -44,32 +76,989 @@ a|
 [source, graphql, indent=0]
 ----
 type Movie {
+  title: String!
+}
+----
+a|
+[source, graphql, indent=0]
+----
+type Movie @node {
+  title: String!
+}
+----
+|===
+
+
+=== Removed the deprecated `options` argument
+
+The deprecated `options` argument has been removed.
+
+Consider the following type definitions:
+
+[source, graphql, indent=0]
+----
+type Movie @node {
+  title: String!
+}
+----
+
+The migration is as below:
+
+[cols="1,1"]
+|===
+| Before | After
+
+a|
+[source, graphql, indent=0]
+----
+{
+  movies(options: { first: 10, offset: 10, sort: [{ title: ASC }] }) {
+    title
+  }
+}
+----
+a|
+[source, graphql, indent=0]
+----
+{
+  movies(first: 10, offset: 10, sort: [{ title: ASC }]) {
+    title
+  }
+}
+----
+|===
+
+The `deprecatedOptionsArgument` of `excludeDeprecatedFields` has been removed.
+
+=== Removed the deprecated `directed` argument
+
+The deprecated `directed` argument has been removed from queries. This argument previously allowed you to specify whether relationships should be directed or undirected at query time.
+
+Please use the `queryDirection` argument of the `@relationship` directive instead.
+
+The `directedArgument` setting of `excludeDeprecatedFields` has been removed.
```

Comment:
```suggestion
The `directedArgument` option of `excludeDeprecatedFields` has been removed.
```

No strong opinion on whether or not we use "option", or "setting" or "key", but we should be consistent with whatever we decide on.